mso/resource_mso_schema_site_vrf_region.go

```diff
@@ -519,7 +530,7 @@ func resourceMSOSchemaSiteVrfRegionRead(d *schema.ResourceData, m interface{}) e
 						}
 						apiRegion := models.StripQuotes(regionCont.S("name").String())
 						if apiRegion == stateRegion {
-							d.SetId(apiRegion)
+							d.SetId(fmt.Sprintf("%s/sites/%s/template/%s/vrf/%s/region/%s", schemaId, stateSite, stateTemplate, stateVrf, stateRegion))
```

Comment:
Should this site id not be set to something as below: 

`fmt.Sprintf("%s/sites/%s-%s/anps/%s/epgs/%s/domainAssociations/%s", schemaId, apiSite, apiTemplate, apiAnp, apiEPG, apiDomain)`

Reply:
I set the ID in a format which is equivalent to the ID we provide for import?

Reply:
We need to discuss this and finalize a format for all the resources as it's done differently in different resources.
I'm resolving this for now.